### PR TITLE
vulkaninfo: Remove invalid json for present_support 

### PR DESCRIPTION
**Pull request by krOoze**
per-discussion in #160

### CODE REVIEW

**Comment by mikew-lunarg** [comment]
@krOoze It would help a lot to create an issue presenting the proposed JSON, which is difficult to extrapolate from the C++ code.  We can discuss there any schema requirements or validation failures.

Samples of the JSON output and the validation error messages are also helpful.

**Comment by krOoze** [comment]
@mikew-lunarg #160

File: vulkaninfo/vulkaninfo.c [hunk 1]
```diff
@@ -4558,11 +4558,6 @@ static void AppGpuDumpQueueProps(const struct AppGpu *gpu, uint32_t id, FILE *ou
         printf("\t\t\t\"queueCount\": %u,\n", props.queueCount);
         printf("\t\t\t\"queueFlags\": %u,\n", props.queueFlags);
         printf("\t\t\t\"timestampValidBits\": %u,\n", props.timestampValidBits);
```
Comment (jeremyk-lunarg) [inline vulkaninfo/vulkaninfo.c]: The comma after `timestampValidBits\": %u` makes this invalid JSON.

Comment (krOoze) [reply vulkaninfo/vulkaninfo.c]: Fixed. **all** validators seem to pass now.

**Pull request merged**